src/mqueryfront/src/QueryResultsStatus.js

```diff
@@ -257,6 +268,12 @@ class QueryResultsStatus extends Component {
                         >
                             {this.props.job.status}
                         </span>
+                        {this.props.job.files_errored > 0 && (
+                            <span style={{ color: "grey", fontSize: "12px" }}>
+                                {" "}
+                                {this.props.job.files_errored} files was missed
```

Comment:
`*were` missed.

Or maybe just something like `xxx errors encountered` / `xxx errors during processing`?

Reply:
Also, maybe move it to the right of processing instead?

Also, not sure why but I have this (0 files)

![ss](https://user-images.githubusercontent.com/7026881/81051502-0f513000-8ec2-11ea-99aa-81cffea4b043.png)

Reply:
Nvm the screenshot, I had the older version.
Still think that maybe it should be moved to the right of `processed`, and maybe displayed in red instead of being gray?

Reply:
Red color looked bad there, gray nicely entered the UI.
I was also thinking about putting it there (the right of processed).